doc/changes/changes_4.0.0.md

```diff
@@ -3,3 +3,6 @@ This release supports specifying individual test files for matrix builds.
 
 ## Features
  - #119: Support single files in flavor config ci.json
+
+## Refactoring
```

Comment:
not really a refactoring, more a bug fix

Reply:
Pls put the change entry into the "unreleased.md", not the changelog for an existing release.
(The way how changelogs work is different here than in the script-languages-release repository).